docs/tools/prepare_examples_tests.py

```diff
@@ -54,8 +54,12 @@
             os.unlink(test_example_file)
             continue
 
-        with open(example_file, "r", encoding="utf-8") as f:
-            lines = f.read().split("\n")
+        try:
+            with open(example_file, "r", encoding="utf-8") as f:
+                lines = f.read().split("\n")
+        except FileNotFoundError:
```

Comment:
what is this for? when does this happen?

Reply:
something (I think Airflow) creates `logs` folder in the examples folder when I run any example. this prevents me from refreshing examples and I must remove the `logs` folder each time